app/src/main/java/org/fossasia/susi/ai/adapters/recyclerAdapters/ChatFeedRecyclerAdapter.java

```diff
@@ -333,18 +334,19 @@ private void handleItemEvents(final MapViewHolder mapViewHolder, final int posit
         final String toSpeak = model.getContent();
         String[] parts = toSpeak.split(":");
         final String string1 = parts[0];
-        if(MainActivity.checkSpeechOutputPref()){
```

Comment:
We need to check whether the user has enabled the speech output from susi or now, so don't remove this check.